api/src/main/java/jakarta/data/page/package-info.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * <p>Splits query results into slices or pages.</p>
```

Comment:
shall we include what is pagination for Jakarta Data here and why it is important?

Reply:
> shall we include what is pagination for Jakarta Data here and why it is important?

That's a good idea.  I added it in https://github.com/jakartaee/data/pull/261/commits/ee9f257d02da1439ad043777b3b81d8ac2f5b64f